### ISSUE TITLE
About the training model

### ISSUE THREAD

**Issue by xiaoyi-st** [google-research/simclr#57]
hi,Thank you for providing such a good training program model.
I have a little question to ask you during the training。What's the biggest difference between Pretrain 'and' Finetune。

**Comment by chentingpc** [comment]
Pre-training is unsupervised (no labels), and fine-tuning is supervised (both images and labels).

**Comment by xiaoyi-st** [comment]
thinks
